### PR TITLE
Fix download_location option of add_torrent

### LINKED ISSUE #8: Download location is overwritten at add a new torrent
I noticed that at add a new torrent selecting a download location as mentioned in the [quickstart](https://github.com/idlesign/deluge-webapi/blob/master/docs/source/quickstart.rst), the download location is overwritten by the one in the deluge configuration, how you can se here [webapi/webui.py](https://github.com/idlesign/deluge-webapi/blob/6008722a3b1718f395b8b32eee2a026925a5bed4/webapi/webui.py#L70).

### PR DESCRIPTION
**Pull request by dmarcosl**
closes #8

### CODE REVIEW

File: webapi/webui.py [hunk 1]
```diff
@@ -67,7 +67,9 @@ def add_torrent(self, metainfo, options=None):
             options = {}
 
         coreconfig = component.get('CoreConfig')
-        options.update({'download_location': coreconfig.get("download_location")})
+
+        if 'download_location' not in options.keys() or options.get('download_location') == 'download_location':
```
Comment (idlesign) [inline webapi/webui.py]: Thank you. I have no access to deluge now, so I'd like to ask to to recheck one thing.
This assumption seems strange: `options['download_location'] == 'download_location'`. Is it indeed how it should be?

**Comment by dmarcosl** [comment]
I added it just in case because at the first quick view of the code I saw that the parameter "options" could have that value and could give a path error, but after reviewing it I think I had looked it bad. I just eliminated the condition.

File: webapi/webui.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def add_torrent(self, metainfo, options=None):
 
         coreconfig = component.get('CoreConfig')
 
-        if 'download_location' not in options.keys() or options.get('download_location') == 'download_location':
+        if 'download_location' not in options.keys():
```
Comment (idlesign) [inline webapi/webui.py]: .keys may be omitted.

**Comment by idlesign** [comment]
Thank you. Merged.

**Pull request merged**